lib/if.h

```diff
@@ -143,9 +143,11 @@ struct if_stats {
 #define TE_EXT_MASK             0x0FFFFFFF
 #define TE_EXT_ANORMAL          0x80000000
 #define LOSS_PRECISION          0.000003
+/* TE_MEGA_BIT and TE_BYTE are utilized to convert TE bandwidth */
 #define TE_MEGA_BIT             1000000
 #define TE_BYTE                 8
-#define DEFAULT_BANDWIDTH       10000
+
+#define DEFAULT_BANDWIDTH 10 /* Default TE bandwidth when no value in conf */
```

Comment:
Please add a comment to said that DEFAULT_BANDWIDTH will be multiply by TE_MEGA_BIT which gives a default value of 10 Mbit/s to avoid someone change the value again

Reply:
This is not what we have decided on slack